deploy-board/deploy_board/templates/clusters/replace_histories.tmpl

```diff
@@ -0,0 +1,37 @@
+{% load utils %}
+{% load static %}
+<table id="deployHistoryTableId" class="table table-condensed table-striped table-hover">
+    <tr>
+        <th>Start(-08:00)</th>
```

Comment:
yea somehow the default page when user clicks on `capacity` is this `cluster configuration page`. (currently pause/cancel/replace cluster buttons are all inside `capacity`
Not sure if I should just move all the actions related with `replacement` outside of `capacity` ?  and add a `replacement` button below `capacity` for all replacements related actions?